docs/userguide/targets.md

```diff
@@ -1,18 +1,87 @@
+experiment and the machine learning core behind BayBE.
+In general, it is expected that you create one [`Target`](baybe.targets.base.Target)
+object for each of your observables.
+The way BayBE treats multiple targets is then controlled via the 
+[`Objective`](../../userguide/objective).
 
-For BaybBE, targets play a crucial role in modeling the function that requires optimization during the DoE campaign.
-Targets are used to model the function that should be optimized by the DoE campaign. They are implemented via the [`Target`](baybe.targets.base.Target) class.
-Each target has a name as well as an abstract transform function. This function is used to transform data into computational representation. 
+## NumericalTarget
+Besides the `name`, a [`NumericalTarget`](baybe.targets.numerical.NumericalTarget)
+has the following attributes:
+* **The optimization** `mode`: This specifies whether we want to minimize or maximize
+  the target or whether we want to match a specific value.
+* **Bounds:** Define `bounds` that constrain the range of target values.
+* **A transformation function:** When bounds are provided, a `transformation` is
+  used to map target values onto the [0, 1] interval.
 
-BayBE currently supports a single type of target, called [`NumericalTarget`](baybe.targets.numerical.NumericalTarget), specifically designed for numerical modeling.
+Below is a visualization of possible choices for `transformation`, where `lower` and
+`upper` are the entries provided via `bounds`:
+![Transforms](../_static/target_transforms.svg)
 
-## Numerical targets
+### MIN and MAX mode
+Here are two examples for simple maximization and minimization targets:
+```python
+from baybe.targets import NumericalTarget, TargetMode, TargetTransformation
 
-Besides their name, numerical targets have the following attributes.
-* **The optimization mode:** This mode dictates whether we want to minimize or maximize the target or whether we want to match a specific value.
-* **Bounds:** Define optional bounds that constrain the range of target values.
-* **A bound transformtaion function:** When bounds are provided, a bound transformation function is used to map target values onto the [0,1] interval. This is only applicable in minimization and maximization mode.
+max_target = NumericalTarget(
+    name="Target_1",
+    mode=TargetMode.MAX,  # can also be provided as string "MAX"
+)
 
-Targets are used on all of the examples, mostly in either minimization or maximization mode. For an example on how the matching mode can be effectively utilized, we refer to the dedicated example provided [here](./../../examples/Multi_Target/desirability).
\ No newline at end of file
+min_target = NumericalTarget(
+    name="Target_2",
+    mode="MIN",  # can also be provided as TargetMode.MIN
+    bounds=(0, 100),  # optional
+    transformation=TargetTransformation.LINEAR,  # optional, will be applied if bounds are not None
```

Comment:
its a bit challenging in this line as Id rate the optional comment more important. Instead Ive included it in the other code blocka bout the match targets where the optional comments were redundant anyway, so a comment there was changed to mention the string variant

Reply:
Understand your point and we can leave it as you proposed. But also note that there is always the (perhaps less attractive but flexible) alternative of placing a longer comment e.g. on top in case it would exceed the line length. We do already have that at some other places. Feel free to resolve